Missing whitespace in dashboard widget template causes tslint to fail.

Missing whitespace in generator-vsts-extension\generators\widget\templates\src\main.ts causes tslint to fail with missig whitespace error;

> tslint -c tslint.json 'src/*.ts'
ERROR: src/main.ts[35, 52]: missing whitespace

This subsequently causes yoman to bubble up the error;
npm ERR! vstsextwidget@0.0.0 lint: `tslint -c tslint.json 'src/*.ts'`
npm ERR! Exit status 2

--- a/generators/widget/templates/src/main.ts
+++ b/generators/widget/templates/src/main.ts
@@ -35,7 +35,7 @@
                         });
                 };
                 return {
-                    load: function (widgetSettings){
+                    load: function (widgetSettings) {
 <% if (UseAITelemetry) { %>
 tc.TelemetryClient.getClient(telemetryClientSettings.settings).trackPageView("<%= WidgetId %>.Index");
 <% } %>

